crates/ty_vendored/vendor/typeshed/stdlib/builtins.pyi

```diff
@@ -707,7 +707,7 @@ class bytes(Sequence[int]):
     def strip(self, bytes: ReadableBuffer | None = None, /) -> bytes: ...
     def swapcase(self) -> bytes: ...
     def title(self) -> bytes: ...
-    def translate(self, table: ReadableBuffer | None, /, delete: bytes = b"") -> bytes: ...
+    def translate(self, table: ReadableBuffer | None, /, delete: ReadableBuffer = b"") -> bytes: ...
```

Comment:
I ran the typeshed sync workflow manually to get this change in, to prevent new false positives in https://github.com/astral-sh/ruff/pull/19142